recipe/meta.yaml

```diff
@@ -11,6 +11,8 @@ source:
 
 build:
     number: {{ buildnumber }}
+    ignore_run_exports:
+      - blas
```

Comment:
@oleksandr-pavlyk, this should be added as this package only depends on mkl and is not tied to the blas implementation.

Reply:
I will make similar changes to mkl_fft-feedstock/  and mkl_random-feedstock/. Thanks